packages/openapi-v3/src/__tests__/unit/spec-contributor/fixtures/application.ts

```diff
@@ -0,0 +1,23 @@
+import {Application, createBindingFromClass} from '@loopback/core';
+import {OAISpecEnhancerService, OAISPEC_ENHANCER_SERVICE} from '../../../../';
```

Comment:
Why do we use `OAISpec` instead of `OpenAPISpec` or `OASpec`?  What would the `I` mean in the acronym? Isn't the `I` part of the other acronym `API`?

Reply:
Agreed, double checked with https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#introduction, I think a more accurate abbreviation should be `OAS`, which stands for OpenAPI Specification. 

I used `OAI` because it's their github organization name.